CanHazFunny/CanHazFunny/Jester.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+    public class Jester
+    {
+        private IJokeService JokeService;
+        private IJokeOutput JokeOutServ;
```

Comment:
Field should be named with underscore PascalCase
```suggestion
        private IJokeService _JokeService;
        private IJokeOutput _JokeOutServ;
```